src/BackgroundEffects.js

```diff
@@ -30,68 +30,30 @@ const growingStars = require('./effects/background/growingStars');
 const squiggles = require('./effects/background/squiggles');
 const musicWave = require('./effects/background/musicWave');
 
-// foreground
-const rain = require('./effects/foreground/rain');
-const rainingTacos = require('./effects/foreground/rainingTacos');
-const pineapples = require('./effects/foreground/pineapples');
-const spotlight = require('./effects/foreground/spotlight');
-const colorLights = require('./effects/foreground/colorLights');
-const smilingPoop = require('./effects/foreground/smilingPoop');
-const heartsRed = require('./effects/foreground/heartsRed');
-const heartsColorful = require('./effects/foreground/heartsColorful');
-const floatingRainbows = require('./effects/foreground/floatingRainbows');
-const bubbles = require('./effects/foreground/bubbles');
-const explodingStars = require('./effects/foreground/explodingStars');
-const pizzas = require('./effects/foreground/pizzas');
-const smileFace = require('./effects/foreground/smileFace');
-const confetti = require('./effects/foreground/confetti');
-const musicNotes = require('./effects/foreground/musicNotes');
-const paintDrip = require('./effects/foreground/paintDrip');
-const emojis = require('./effects/foreground/emojis');
-
-module.exports = class Effects {
-  constructor(p5, alpha, extraImages, blend, currentPalette = 'default') {
+module.exports = class BackgroundEffects {
+  constructor(p5, alpha, getEffectsInPreviewMode, extraImages) {
```

Comment:
If you keep foreground and background effects in different files, what do you think of not passing `alpha` as a parameter and defining it as a constant such as `COLOR_OPACITY` and referencing it in `colorFromHue`?